clang/test/CodeGenSYCL/add_ir_attributes_function.cpp

```diff
@@ -137,8 +137,8 @@ class KernelFunctor6 {
 public:
 #ifdef __SYCL_DEVICE_ONLY__
   [[__sycl_detail__::add_ir_attributes_function(
-      "", "Prop12", "", "", "", "Prop16", "Prop17",
-      "Another property string", 2, false, TestEnum::Enum1, nullptr, nullptr, ScopedTestEnum::ScopedEnum2)]]
+      "", "Prop12", "", "", "", "Prop16", "Prop17", "Prop18",
```

Comment:
Just for my education, can we attach properties to a kernel if it is defined as a lambda?

Reply:
Absolutely! This is done through the new `parallel_for`/`single_task`/... overloads. We apply this attribute to the known kernel function (e.g. `kernel_single_task`) and it is then propagated to the kernel function definition.